shelter/permanent-housing-found.vbs

```diff
@@ -51,150 +52,198 @@ changelog_display
-		IF IsDate(move_date) = False then err_msg = err_msg & vbNewLine & "* Enter a numeric move in date."
-		If monthly_rent = "" then err_msg = err_msg & vbNewLine & "* Enter the move in date."
-		If vendored_to_HCEA = "" then err_msg = err_msg & vbNewLine & "* Enter the amt vendored to HCEA shelter acct."
-		If shelter_cost = "" then err_msg = err_msg & vbNewLine & "* Enter the shelter costs."
-		If isNumeric(num_nights) = False then err_msg = err_msg & vbNewLine & "* Enter a numeric number of nights at the shelter."
-		If isNumeric(balance_HCEA_acct) = False then err_msg = err_msg & vbNewLine & "* Enter the HCEA Shelter account balance available for rent/DD."
-		If isNumeric(rent_needed) = False then err_msg = err_msg & vbNewLine & "* Enter a numeric rent amount needed to move in."
-		If isNumeric(DD_needed) = False then err_msg = err_msg & vbNewLine & "* Enter a numeric DD amount needed to move in."
-		If isNumeric(client_funds) = False then err_msg = err_msg & vbNewLine & "* Enter a numeric amount of client funds available."
-		If rent_subsidy_paid = "" then err_msg = err_msg & vbNewLine & "* Enter who will be paying for the rental subsidy."
-		If isNumeric(rent_subsidy) = False then err_msg = err_msg & vbNewLine & "* Enter a numeric subsidy amount."
-		If isNumeric(vendor_to_LL) = False then err_msg = err_msg & vbNewLine & "* Enter a numeric amount for the balance in HCEA shelter acct that will be vendored to LL."
-		If additional_funds = "" then err_msg = err_msg & vbNewLine & "* Enter who will be issuing the additional funds."
-		If LL_name = "" then err_msg = err_msg & vbNewLine & "* Enter the Landlord's name."
-		If LL_ADDR = "" then err_msg = err_msg & vbNewLine & "* Enter the Landlord's address."
-		If ESP = "" then err_msg = err_msg & vbNewLine & "* Enter the name of the Employment Services Provider."
-		If REW = "" then err_msg = err_msg & vbNewLine & "* Enter the name of the Rapid Exit Worker."
-		If isdate(closed_servicepoint) = False then err_msg = err_msg & vbNewLine & "* Enter the date that ServicePoint was closed out."
-		If isdate(mand_vend_date) = False then err_msg = err_msg & vbNewLine & "* Enter the date the mandatory vendor status changed."
-		If client_phone = "" then err_msg = err_msg & vbNewLine & "* Enter the client's phone number."
-		If isNumeric(TANF_months) = False then err_msg = err_msg & vbNewLine & "* Enter numeric number of TANF months used."
-		If isNumeric(num_days_shelter) = False then err_msg = err_msg & vbNewLine & "* Enter the numeric number of days in shelter."
-		IF worker_signature = "" THEN err_msg = err_msg & vbCr & "* Sign your case note."
-		IF err_msg <> "" THEN MsgBox "*** NOTICE!!! ***" & vbNewLine & err_msg & vbNewLine
-	LOOP UNTIL err_msg = ""
+	  DO
+      err_msg = ""
+      Dialog Dialog1
+      cancel_confirmation
+      IF IsDate(move_date) = False then err_msg = err_msg & vbNewLine & "* Enter a numeric move in date."
```

Comment:
For error handling for the dates, I think it might be clearer to say something along the lines of "Enter a move in date in the format MM/DD/YYYY" instead of numeric date.